app/views/home/_listing.html.erb

```diff
@@ -0,0 +1,9 @@
+<% @users.each do |user| %>
+  <tr>
+    <td><%= user.email %></td>    
+    <td><%= user.has_gender? %></td>
```

Comment:
There is a good practice that method ending with the question mark should return a boolean value, which is not the case. Consider renaming methods to follow aforementioned best practice. In general, it's ok to name method `gender` and leave the responsibility to return something meaningful to the decorated object.